contributors/design-proposals/storage/volume-topology-scheduling.md

```diff
@@ -927,11 +926,12 @@ BindPodVolumes(pod *v1.Pod, node *v1.Node) (err error)
     2. If the prebind fails, revert the cache updates.
 2. For in-tree and external dynamic provisioning:
     1. Set `annSelectedNode` on the PVC.
-3. Send Pod back through scheduling, regardless of success or failure.
-    1. In the case of success, we need one more pass through the scheduler in
-order to evaluate other volume predicates that require the PVC to be bound, as
-described below.
-    2. In the case of failure, we want to retry binding/provisioning.
+3. Wait for binding and provisioning to complete.
+    1. In the case of failure, error is returned and the Pod will retry
```

Comment:
also add a line in the picture to indicate rescheduling on failure?

Reply:
A failure at any step will trigger rescheduling.  It might be too messy to include in the diagram.

Reply:
that makes sense